umap/static/umap/js/umap.features.js

```diff
@@ -483,12 +483,30 @@ L.U.FeatureMixin = {
     return false
   },
 
+  parseDateField: function (value) {
+    if (parseFloat(value).toString() === value.toString()) {
```

Comment:
`value` may be undefined, which would make `parseFloat` to fail. As a general advice, given the data uMap will load can come from user inputs and from remote source, we have to be very defensive. The feature may not work, but the map should not crash, so the user can fix the data or settings.

Reply:
You are absolutely right. Actually, I already saw that problem shortly after this pull request and resolved it on the spot. It's part of this commit: https://github.com/umap-project/umap/pull/1491/commits/96f808383291b6b880752e67f1e53bb349018999